schema/install_schema.go

```diff
@@ -7,65 +7,65 @@ import (
 // InstallSchema represents the install block in the Kairos configuration. It is used to drive automatic installations without user interaction.
 type InstallSchema struct {
 	_                   struct{}       `title:"Kairos Schema: Install block" description:"The install block is to drive automatic installations without user interaction."`
-	Auto                bool           `json:"auto,omitempty" description:"Set to true when installing without Pairing"`
-	BindMounts          []string       `json:"bind_mounts,omitempty"`
-	Bundles             []BundleSchema `json:"bundles,omitempty" description:"Add bundles in runtime"`
-	NoFormat            bool           `json:"no_format,omitempty"`
-	Device              string         `json:"device,omitempty" pattern:"^(auto|/|(/[a-zA-Z0-9_-]+)+)$" description:"Device for automated installs" examples:"[\"auto\",\"/dev/sda\"]"`
-	EphemeralMounts     []string       `json:"ephemeral_mounts,omitempty"`
-	EncryptedPartitions []string       `json:"encrypted_partitions,omitempty"`
-	Env                 []interface{}  `json:"env,omitempty"`
-	GrubOptionsSchema   `json:"grub_options,omitempty"`
-	Image               string `json:"image,omitempty" description:"Use a different container image for the installation"`
+	Auto                bool           `json:"auto,omitempty" description:"Set to true when installing without Pairing" yaml:"auto,omitempty"`
+	BindMounts          []string       `json:"bind_mounts,omitempty" yaml:"bind_mounts,omitempty"`
+	Bundles             []BundleSchema `json:"bundles,omitempty" description:"Add bundles in runtime" yaml:"bundles,omitempty"`
+	NoFormat            bool           `json:"no_format,omitempty" yaml:"no_format,omitempty"`
+	Device              string         `json:"device,omitempty" pattern:"^(auto|/|(/[a-zA-Z0-9_-]+)+)$" description:"Device for automated installs" examples:"[\"auto\",\"/dev/sda\"]" yaml:"device,omitempty"`
+	EphemeralMounts     []string       `json:"ephemeral_mounts,omitempty" yaml:"ephemeral_mounts,omitempty"`
+	EncryptedPartitions []string       `json:"encrypted_partitions,omitempty" yaml:"encrypted_partitions,omitempty"`
+	Env                 []interface{}  `json:"env,omitempty" yaml:"env,omitempty"`
+	GrubOptionsSchema   `json:"grub_options,omitempty" yaml:"grub_options,omitempty"`
+	Image               string `json:"image,omitempty" description:"Use a different container image for the installation" yaml:"image,omitempty"`
```

Comment:
I don't think this one has a direct mapping to the install struct

Reply:
@nianyush we do not have Image in the Install struct, kindly advise if we'd need that for our use-case